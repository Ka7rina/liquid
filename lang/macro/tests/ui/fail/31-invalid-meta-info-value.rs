use liquid_lang as liquid;

#[liquid::contract(version = "0.1.0", hash_type = "sm4")]
mod noop {
    #[liquid(storage)]
    struct Noop {}

    #[liquid(methods)]
    impl Noop {
        pub fn new(&mut self) {}

        pub fn noop(&self) {}
    }
}

fn main() {}
