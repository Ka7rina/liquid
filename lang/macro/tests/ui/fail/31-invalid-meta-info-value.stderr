error: invalid hash type: sm4
 --> $DIR/31-invalid-meta-info-value.rs:3:51
  |
3 | #[liquid::contract(version = "0.1.0", hash_type = "sm4")]
  |                                                   ^^^^^
